schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR
```

Comment:
- You need to run an update command to change the schema of the Animals table.
Something like below will do:
`ALTER TABLE animals ADD species varchar(255);
`